website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -6,115 +6,63 @@ weight: 3
 
 This guide walks through an example of building a simple memcached-operator powered by Ansible using tools and libraries provided by the Operator SDK.
 
-## Create a new project
+## Prerequisites
 
-After [installing the Operator SDK CLI][install-guide] and
-[ansible operator prerequisites][ansible-install-guide], use the CLI to create a
-new Ansible-based memcached-operator project:
+- [docker][docker-tool] version 17.03+.
+- [kubectl][kubectl-tool] version v1.11.3+.
+- [Ansible Operator SDK Installation][ansible-operator-install] v1.0.0+
+- Access to a Kubernetes v1.11.3+ cluster.
```

Comment:
```suggestion
- [Install `operator-sdk`][operator_install] and the [Ansible prequisites][ansible-operator-install] 
- Access to a Kubernetes v1.16.0+ cluster.
```

Reply:
Motivations:
- Otherwise, you will unable to apply the CRDs. They are using v1. 
- Less verbose.